tests/utils/tracked_container.py

```diff
@@ -19,22 +19,18 @@ class TrackedContainer:
         Docker client instance
     image_name: str
         Name of the docker image to launch
-    **kwargs: dict, optional
```

Comment:
We only construct TrackedContainer in one place, it doesn't make sense to allow this customization, and we simply add `detach=True` in `run_detached`